src/containers/Header/Header.js

```diff
@@ -6,12 +6,34 @@ import ThemeSwitch from '../../components/ThemeSwitch';
 
 import './Header.css';
 
-const Header = () => (
-  <div className="header">
-    <Sidebar />
-    <LocaleToggle />
-    <ThemeSwitch />
-  </div>
-);
+class Header extends React.Component {
+  state = {
+    showSidebar: false,
+  };
+
+  sidebarClosedHandler = () => {
+    this.setState({ showSidebar: false });
+  };
+
+  sidebarToggleHandler = () => {
+    this.setState(prevState => {
+      return { showSidebar: !prevState.showSidebar };
+    });
+  };
+
+  render() {
+    return (
+      <div className="header">
+        <Sidebar
```

Comment:
I do not really think that sidebar should be a child of header, but it should be on the same level as header, footer and main content.